ecosystem/sep-0010.md

```diff
@@ -187,6 +206,20 @@ Every other HTTP status code will be considered an error. For example:
 }
 ```
 
+## JWT Expiration
+
+Servers should select an expiration time for the JWT that is appropriate for the assumptions and risk of the interactions a user can perform with it. A user may be in control of an account at the time the JWT is issued but they may lose control of the account through changes in signers.
```

Comment:
@msfeldstein @tomquisel I added a statement about expiration and removed the 24 hour recommendation we had before. I don't think this is something we should define for services. We might want to define it for specific types of services, like an anchor, but not sure we should do that generically. What do you think?

Reply:
yeah i think this is right.  I don't think we can codify something that's this subjective.

Reply:
This makes sense to me. I'd also mention that a user may have to go through an inconvenient process to get a JWT (say use a Ledger Nano or use StellarGuard), so too short an expiration time could lead to poor UX.

Reply:
@tomquisel I've added words about that in e1800e4c3d6b46e006e88922abfd961e73c73bb2.